website/snippets/_cloud-environments-info.md

```diff
@@ -34,6 +34,16 @@ Both development and deployment environments have a section called **General Set
 - If you select a current version with `(latest)` in the name, your environment will automatically install the latest stable version of the minor version selected.
 :::
 
+### Repo caching <Lifecycle status='Enterprise plan' /> 
+
+At the start of every job run, dbt Cloud clones the project's Git repository so it has the latest versions of the project code and package code. 
```

Comment:
I like the enterprise tag, but feel like we should also call out enterprise only as a note.

Reply:
fair point! i'll use a callout to mark this feature as "enterprise only" and remove the pill